app/adapters/event.js

```diff
@@ -0,0 +1,19 @@
+import ApplicationAdapter from './application';
+import ENV from 'open-event-frontend/config/environment';
+import { computed } from '@ember/object';
+
+export default ApplicationAdapter.extend({
+
+  headers: computed('session.data.authenticated', 'session.currentRouteName', function() {
```

Comment:
Use `router.currentRouteName` instead of the property from session.

https://emberjs.com/api/ember/3.2/classes/RouterService/properties/currentRouteName?anchor=currentRouteName